src/PropertySuggester/GetSuggestions.php

```diff
@@ -121,6 +121,7 @@ private function querySearchApi( $resultSize, $search, $language ) {
 				'search' => $search,
 				'action' => 'wbsearchentities',
 				'language' => $language,
+				'uselang' => $language,
```

Comment:
This is an intended change, see [T109584](https://phabricator.wikimedia.org/T109584), but probably should be in it's own patch.